cli/format.go

```diff
@@ -69,6 +69,40 @@ func (f *Format) Run() (err error) {
 		}
 	}()
 
+	// find the config file unless specified
+	if Cli.ConfigFile == "" {
+		pwd, err := os.Getwd()
+		if err != nil {
+			return err
+		}
+		Cli.ConfigFile, _, err = findUp(pwd, "treefmt.toml")
+		if err != nil {
+			return err
+		}
+	}
+
+	// search for the project root unless specified
+	if Cli.TreeRoot == "" {
+		// use the location of the treefmt.toml file by default
```

Comment:
I think this could be refined if the `xor` group is used for the flags

Reply:
I don't see how

Reply:
enlightenement gif